Kernel/API/Syscall.h

```diff
@@ -58,7 +58,7 @@ enum class NeedsBigProcessLock {
     S(clock_settime, NeedsBigProcessLock::No)               \
     S(close, NeedsBigProcessLock::No)                       \
     S(connect, NeedsBigProcessLock::No)                     \
-    S(create_inode_watcher, NeedsBigProcessLock::Yes)       \
+    S(create_inode_watcher, NeedsBigProcessLock::No)        \
```

Comment:
In this syscall, there's a problem with how allocation of a fd happens:
https://github.com/SerenityOS/serenity/blob/fa637e29d2fb1c9d161e204e4e511529f795207d/Kernel/Syscalls/inode_watcher.cpp#L21-L22
Look at how it is done in `anon_create` syscall:
https://github.com/SerenityOS/serenity/blob/fa637e29d2fb1c9d161e204e4e511529f795207d/Kernel/Syscalls/anon_create.cpp#L39-L43

We should ensure it happens here too.